Vwitter/Models/VWUser.m

```diff
@@ -0,0 +1,14 @@
+//
+//  VWUser.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/8/22.
+//
+
+#import "VWUser.h"
+
+@implementation VWUser
+
+@dynamic screenName;
```

Comment:
I am a bit skeptical of how `VWUser` is being used, does it ever make sense for us to initialize a `VWUser` without a screenname? If so, you should probably have a custom `initWithScreenname` method, like what you are doing with `Follow`